src/status.rs

```diff
@@ -483,6 +535,12 @@ impl StatusBuilder {
             url: self.url.ok_or("url missing")?,
             location: self.location.ok_or("location missing")?,
             contact: self.contact.ok_or("contact missing")?,
+            spacefed: self.spacefed,
+            projects: self.projects,
+            cam: self.cam,
+            feeds: self.feeds,
+            events: self.events,
+            radio_show: self.radio_show,
```

Comment:
Yes :) Well, there is [#[derive(Builder)]](https://crates.io/crates/derive_builder), but afaict it does not support methods like `add_event()` yet, which append to a vector while building.